params/config.go

```diff
@@ -359,6 +368,7 @@ var (
 		BanffBlockTimestamp:             utils.NewUint64(0),
 		CortinaBlockTimestamp:           nil,
 		DurangoBlockTimestamp:           nil,
+		EUpgradeTime:                    nil,
```

Comment:
nit: I think in GoLang providing zero-values that already have default behavior can be omitted, so I am in favor of removing all these `...Timestamp: nil` lines

Reply:
sounds good